core/src/components/input/input.scss

```diff
@@ -103,7 +104,7 @@
 }
 
 .native-input[disabled] {
-  opacity: .4;
+  opacity: var(--disabled-opacity, .4);
```

Comment:
It would be great if we could make this change without adding a new variable. 

I noticed you also had another PR that did something similar: https://github.com/ionic-team/ionic/pull/18509/files#diff-7c58c82ca091ee01c6244994d0362292R65

Is there a reason why we can't do that here?

Reply:
Yes, you're right. I'll change it. Thanks!

I wasn't sure of which one to use (actually, I edited the issue a couple of times in regards to that). At first I tried with --disabled-opacity to clearly identify that this variable would only affect the disabled state.